src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
Wouldn't this be wrong? 

we check for `Illuminate\Foundation\Application` because these commands only works on Laravel, not Lumen.